cli_reference/basic_cli_operations.adoc

```diff
@@ -134,63 +136,63 @@ The following table describes basic `oc` operations and their general syntax:
 |Operation |Syntax |Description
 
 |`get`
-|`oc get _<object_type>_ [_<object_name_or_id>_]`
+|`oc get <object_type> [<object_name_or_id>]`
 |Return a list of objects for the specified link:#object-types[object type]. If
 the optional `_<object_name_or_id>_` is included in the request, then the list
 of results is filtered by that value.
 
 |`describe`
-|`oc describe _<object_type>_ _<object_id>_`
+|`oc describe <object_type> <object_id>`
 |Returns information about the specific object returned by the query. A specific
 `_<object_name_or_id>_` must be provided. The actual information that is
 available varies as described in link:#object-types[object type].
 
 .3+|`edit`
-|`oc edit _<object_type>_/_<object_type_name>_`
+|`oc edit <object_type>/<object_type_name>`
 |Edit the desired object type.
 
-|`OSC_EDITOR="_<text_editor>_" oc edit \`
-`_<object_type>_/_<object_type_name>_`
+|`OSC_EDITOR="<text_editor>" oc edit \`
+`<object_type>/<object_type_name>`
 |Edit the desired object type with a specified text editor.
 
-|`oc edit _<object_type>_/_<object_type_name>_ \`
-`--output-version=_<object_type_version>_ \`
-`-o _<object_type_format>_`
+|`oc edit <object_type>/<object_type_name> \`
+`--output-version=<object_type_version> \`
+`-o <object_type_format>`
 |Edit the desired object in a specified format (eg: JSON).
 
 |`env`
-|`oc env _<object_type>_/_<object_type_name>_ \`
-`_<EN_VAR>_=/_<VALUE>_`
-|Update the desired object type with a new environment variable
+|`oc env <object_type>/<object_type_name> \`
+`<EN_VAR>_=/_<VALUE>`
```

Comment:
I think this can be:

```
`<var_name>=<value>`
```

To match "var_name" used elsewhere in the topic, and to remove italics and what appears to be a stray `/`.